src/RegisterService.Tests/Unit/OrdersController.Test.cs

```diff
@@ -0,0 +1,83 @@
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using ProdtestApi.Services;
+using ProdtestApi.Controllers;
+using ProdtestApi.Contracts;
+using ProdtestApi.Models;
+
+namespace ProdtestApi.Test.Unit;
+
+[TestClass]
+public class OrderControllerTests
+{
+    private Mock<IKitchenService> mock { get; } = new Mock<IKitchenService>();
+
+    [TestMethod]
+    public void Get_OrdersMenuItems_Returns_ListOfMenuItems()
+    {
```

Comment:
Ik zou dit zelf nooit los testen. Hierbij is het altijd goed om te kijken wat de "unit" is in een unit test.